scripts/prepareDeploy.sh

```diff
@@ -105,8 +105,8 @@ for i in $modules
 fi
 
 echo "Deploying "`echo $modules | wc -w`" Modules from branch=$branch to profile=$profile"
-## '.' points to the root project, 'vaadin-flow-components-shared' has the dependencies for demo and tests
-build=.,vaadin-flow-components-shared
+## '.' points to the root project, 'vaadin-flow-components-shared-parent' has the common dependencies for components
+build=.,vaadin-flow-components-shared-parent,vaadin-flow-components-shared-parent/vaadin-flow-components-common
```

Comment:
```suggestion
build=.,vaadin-flow-components-shared-parent,vaadin-flow-components-shared-parent/vaadin-flow-components-base
```
I decided to name the new module to `vaadin-flow-components-base` in #3074

Reply:
oops.. okay,, i will update the name for this module